### PR TITLE
Remove Go 1.9 / AppEngine support; assume go1.12 build tag

### PR DESCRIPTION
**Pull request by dfawley**

### CODE REVIEW

File: internal/credentials/go110.go [hunk 1]
```diff
@@ -1,5 +1,3 @@
-// +build go1.10
```
Comment (menghanl) [inline internal/credentials/go110.go:1]: Rename this file?
Or delete?

File: security/advancedtls/sni_110.go [hunk 2]
```diff
@@ -1,5 +1,3 @@
-// +build go1.10
```
Comment (menghanl) [inline security/advancedtls/sni_110.go:1]: Rename file

File: internal/credentials/go110.go (hunk 1 above)
Comment (dfawley) [reply internal/credentials/go110.go:1]: Renamed.

File: security/advancedtls/sni_110.go (hunk 2 above)
Comment (dfawley) [reply security/advancedtls/sni_110.go:1]: Done

**Comment by dfawley** [comment]
cc @ZhenLian and @yihuazhang as FYI

**Pull request merged**